src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/adaptiveForm/wizard/.content.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
-    jcr:description="Adaptive Form Wizard (v1) component"
+    jcr:description="Wizard - a multi-step forms showing one step at a time"
```

Comment:
is this string reviewed by CHL? seems like a grammatical mistake.

Reply:
reverted to earlier version